1906-12-10.xml

```diff
@@ -0,0 +1,1515 @@
+                                <cell>*Philae</cell>
+                                <cell>Tons 5,000</cell>
+                                <cell>Taber</cell>
+                                <cell>3,000</cell>
+                            </row>
+                        </table>
+                        <p>*Second class accommodation only, unless specially reserved.—Fares :
+                            Alexandria to Liverpool, 1st, £14 Single, £25 Return. 2nd, £9 Single,
+                            £15 Return.—To Malta, 1st, £5 Single, £9 Return, 2nd, £3 Single, £5
+                            Return.—Return tickets available for six months.</p>
+                        <p><name>S.S. Amasia</name> now on the berth, will sail on or about <date
+                                when="1905-07-17">Saturday, December 15</date>, to be followed by
+                                <name>S.S. Menepthah</name></p>
+                        <p>Through freight rates on cotton, etc., to Lancashire inland towns,
+                            Boston, New York and other U.S.A. towns, obtained on application. Cargo
+                            taken by special agreement only.</p>
+                        <p>Passenger Tickets also issued inclusive of Railway fare through to and
+                            from Cairo. Particulars on application to</p>
+                        <p>R. J. MOSS &amp; Co., Alexandria, Agents.</p>
+                        <p><measure type="indexNo">27-11-906</measure></p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-bam02">
+                        <head>Asia Minor Steamship Co., Ltd.</head>
+                        <p>Regular weekly sailings to Cyprus and Coast of Syria. Steamer leaves
+                            Alexandria every Tuesday afternoon for Limassol and Larnaca, returning
+                            viâ Port Said. 1st Class Single Fare, Alex. to Cyprus £2</p>
+                        <p>Sailings liable to alteration owing to Quarantine, etc.</p>
+                        <p>For further particulars apply: Manager, 3 St. Mark's-Street, Alexandria.
+                                <measure type="indexNo">28478-31-12-06</measure></p>
+                    </div>
+                    <!--The next advertisement doesnt match any of the templetes, "Cunard Line"-->
+                </div>
+                <div type="advert" xml:id="deg-ad-wsl02">
+                    <head>WHITE STAR LINE.</head>
+                    <p>SPECIAL WINTER SAILINGS of the twin screw S.S. CEDRIC (21,035 tons)and CELTIC
+                        (20,904 tons)</p>
+                    <p>Between ALEXANDRIA, NAPLES and U.S.A,</p>
+                    <p>First Sailing, Thursday, January 24th, 1907.</p>
+                    <p>For particulars apply: THOS. COOK and SON, CAIRO, and JOHN ROSS and CO.,
+                        ALEXANDRIA</p>
+                    <!--There was no templete so i tried to do it on my own-->
```

Comment:
Do you mean for White Star? Submit an issue if you can and I'll try to use your template to make others.

Reply:
No I was referring to the advertisement below, the Henderson Line one. How do I submit an issue?

Reply:
Go to the advertisements repository and click "issues" in the bar at the top of the page.